tests/ManageCourses.Tests/UnitTesting/Mapping/CourseLoaderTests.cs

```diff
@@ -39,6 +39,41 @@ public void TolerateBlankStartDate()
             res.StartDate.Should().BeNull();
         }
 
+        [Test]
+        public void RunningLocationsArePreferred()
+        {
+            var sut = new CourseLoader();
+            var loc1 = GetBlankUcasCourse();
+            loc1.Status = "S";
+            loc1.AccreditingProvider = "WRONG_ACC";
+            loc1.InstCode = "WRONG_INST";
+
+            var loc2 = GetBlankUcasCourse();
+            loc2.Status = "R";
+            loc2.AccreditingProvider = "RIGHT_ACC";
+            loc2.InstCode = "RIGHT_INST";
+
+            var res = sut.LoadCourse(new List<UcasCourse> {loc1, loc2}, new List<UcasCourseEnrichmentGetModel>(), false);
+
+            res.AccreditingProviderId.Should().Be("RIGHT_ACC");
+            res.InstCode.Should().Be("RIGHT_INST");
+        }
+
+        [Test]
+        public void FullySuspendedCoursesWorkStill()
+        {            
+            var sut = new CourseLoader();
+            var loc1 = GetBlankUcasCourse();
+            loc1.Status = "S";
```

Comment:
what is `S`?

Reply:
"suspended". There are also *D* for discontinued, *N* for new, and of course *R* for running